opslevel/resource_opslevel_service.go

```diff
@@ -101,26 +100,11 @@ func resourceService() *schema.Resource {
 				ForceNew:    false,
 				Optional:    true,
 				Elem:        &schema.Schema{Type: schema.TypeString},
-				// ValidateFunc: validateServiceTags, // TODO: Not Yet Supported
```

Comment:
This `TODO` item isn't valid because `ValidateFunc` does not work on `schema.TypeList`. Source:

```
// ValidateFunc is honored only when the schema's Type is set to TypeInt,
// TypeFloat, TypeString, TypeBool, or TypeMap. It is ignored for all other types.
ValidateFunc [SchemaValidateFunc](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/helper/schema@v1.17.2#SchemaValidateFunc)
```

https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/helper/schema@v1.17.2#Schema.ValidateFunc

Reply:
There already exists examples of adding tags to services that the user can refer to. If the list is invalid they will get an API error anyways.

[Examples](https://registry.terraform.io/providers/OpsLevel/opslevel/latest/docs/resources/service)